### ISSUE TITLE
Marker clustering

### ISSUE THREAD

**Issue by sarahansen223h** [webglearth/webglearth2#53]
Hi,

- We can see that the globe support frustum culling which is nice to have and lower the cpu usage, but somthing like "marker clustering" as  an option in the constuctor would be really nice to have so the markers dont overlap.

Interesting stuff to read from Leaflet ( btw we don't plan to use Leaflet anytime soon ) 

http://leafletjs.com/2012/08/20/guest-post-markerclusterer-0-1-released.html

**Comment by klokan** [comment]
A good idea. It could be done probably on the application level - without a need to change the WebGL Earth API - just by porting the above mentioned or similar cluster manager code.

**Comment by sarahansen223h** [comment]
indeed, any plans or chance that you may patch that one? we don't mind paying for it support in case it need.

**Comment by sarahansen223h** [comment]
any response?

**Comment by klokan** [comment]
We have no support contract with you. Please contact info@klokantech.com with request for what should be done for you, how urgent is it - and we will make you estimate and let you know possible schedule.

Otherwise - feel free to implement what you need yourself - and provide a pull request to this open-source project..

**Comment by sarahansen223h** [comment]
Just sent a mail to: info@klokantech.com with a request.

**Comment by btyukodi** [comment]
Hi, any news on marker clustering?... Thanks!

**Comment by klokan** [comment]
There has been no related contract - and nobody else from the community did the work yet either.

**Comment by overfull** [comment]
The level of WebGLearth is updated for marker clustering, isn't it?

**Comment by avipersin** [comment]
Has anyone gotten marker clustering working?

**Comment by zhaohongda1987** [comment]
Hi, webGl earth is support for marker cluster now? anyone knows or may give me a suggest,thanks.

**Comment by riccardolardi** [comment]
Any update on this?
